web/packages/teleterm/src/ui/DocumentCluster/ClusterResources/Databases/Databases.tsx

```diff
@@ -61,7 +61,7 @@ function DatabaseList(props: State) {
       <SearchPanel
         updateQuery={updateQuery}
         updateSearch={updateSearch}
-        pageCount={pageCount}
+        pageIndicators={pageCount}
```

Comment:
Should `pageCount` now be called `pageIndicators`?

Reply:
This is in `teleterm` and so isn't using the same `SearchPanel` we have in `teleport`, so the prop names aren't the same. We could look into refactoring the `teleterm` implementation of this feature as well in the future to make sure we're reusing components like that, I think doing it here would make this PR too large.